### PR TITLE
docs: Update the maven plugins in the project readme

### PR DESCRIPTION
**Pull request by ST-DDT**
There are a few improvements available in the plugin updates and we should point to the latest version in our readme.

https://github.com/xolstice/protobuf-maven-plugin/compare/protobuf-maven-plugin-0.5.1...protobuf-maven-plugin-0.6.1?expand=1

* Mainly documentation improvements, but also some bugfixes (including Fixed Zip Slip(?))

https://github.com/trustin/os-maven-plugin/releases

* Improved/extended OS detection, better eclipse integration

I use the updated plugins myself and didn't encounter any problems with them yet.

### CODE REVIEW

**Comment by creamsoup** [comment]
thanks for the PR. can you also update the versions in other locations
- [SECURITY.md ](https://github.com/grpc/grpc-java/blob/master/SECURITY.md) 
- [examples/example-gauth/pom.xml](https://github.com/grpc/grpc-java/blob/master/examples/example-gauth/pom.xml)
- [examples/example-tls/pom.xml](https://github.com/grpc/grpc-java/blob/master/examples/example-tls/pom.xml)
- [examples/pom.xml](https://github.com/grpc/grpc-java/blob/master/examples/pom.xml)

**Comment by ST-DDT** [comment]
I'll do the other changes early next week.

**Comment by ST-DDT** [comment]
Updated/Fixed

File: README.md [hunk 1]
```diff
@@ -93,14 +93,14 @@ For protobuf-based codegen integrated with the Maven build system, you can use
     <extension>
       <groupId>kr.motd.maven</groupId>
       <artifactId>os-maven-plugin</artifactId>
-      <version>1.5.0.Final</version>
+      <version>1.6.2.Final</version>
```
Comment (creamsoup) [inline README.md]: remove `.Final` and also elsewhere.

**Comment by creamsoup** [comment]
merged!

**Pull request merged**